src/minter-suite-mapping.ts

```diff
@@ -492,6 +505,255 @@ export function handleDelegationRegistryUpdatedGeneric<T>(event: T): void {
   minter.save();
 }
 
+// MinterSEA specific handlers
+export function handleAuctionDurationSecondsRangeUpdated(
+  event: AuctionDurationSecondsRangeUpdated
+): void {
+  let minter = loadOrCreateMinter(event.address, event.block.timestamp);
+
+  if (!minter) {
```

Comment:
In this case here, and below, should we log a warning within these guards?

If we do that, should we add a utility function?

Reply:
good point - realized we don't even need these null minter guards because `loadOrCreateMinter` never returns null. This was some old habits from when we didn't have the helper function. Updated in 061dd92 🙏 

Note that if we do have guards like these, in general, I would prefer to log a warning to help keep us aware of any ongoing issues in deployed subgraphs!